components/playback/controls/player/DesktopPlayerControls.vue

```diff
@@ -271,8 +332,43 @@ export default {
       "one_min_warning",
       "five_min_warning",
     ]),
-    ...mapState("settings", ["customer_index", "auto_upload", "beta_2_mode"]),
+    ...mapState("settings", [
+      "customer_index",
+      "auto_upload",
+      "beta_2_mode",
+      "user_cred_input_needed",
+      "firmware_update_available",
+      "firmware_update_dur_mins",
+    ]),
+    ...mapState("stimulation", ["stim_status"]),
+    ...mapGetters({
+      status_uuid: "flask/status_id",
+    }),
+    fw_update_available_labels: function () {
+      let duration = `${this.firmware_update_dur_mins} minute`;
+      if (duration !== 1) duration += "s";
```

Comment:
I don't think 'duration' would ever equal 1 in this case

Reply:
Yeah that's definitely a mistake, should be `this.firmware_update_dur_mins !== 1`